src/openrct2/rct1.h

```diff
@@ -29,7 +29,12 @@
 
 #define RCT1_MAX_MAP_ELEMENTS       0xC000
 #define RCT1_MAX_SPRITES            5000
-#define RCT1_MAX_VEHICLES_PER_RIDE  12
+#define RCT1_MAX_STATIONS           4
```

Comment:
This needs to be merged with `RCT2_MAX_STATIONS` as `RCT12_MAX_STATIONS` and be put into `rct12.h`.